src/dashboards/__tests__/__snapshots__/dashboard.test.tsx.snap

```diff
@@ -0,0 +1,424 @@
+              >
+                <div
+                  class="overflow-x-auto"
+                >
+                  <table
+                    class="table-fixed w-full divide-y divide-gray-200 "
+                    role="table"
+                    style="min-width: 120px;"
+                  >
+                    <thead
+                      class="bg-gray-50"
+                    >
+                      <tr
+                        class=""
+                        role="row"
+                        style="display: flex; flex: 1 0 auto; min-width: 120px;"
+                      >
+                        <th
+                          colspan="1"
+                          role="columnheader"
+                          scope="col"
+                          style="box-sizing: border-box; flex: 150 0 auto; min-width: 30px; width: 150px;"
+                        >
+                          <div
+                            class="px-4 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider flex whitespace-nowrap"
+                          >
+                            Name
+                          </div>
+                        </th>
+                        <th
+                          colspan="1"
+                          role="columnheader"
+                          scope="col"
+                          style="box-sizing: border-box; flex: 150 0 auto; min-width: 30px; width: 150px;"
+                        >
+                          <div
+                            class="px-4 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider flex whitespace-nowrap"
+                          >
+                            Status
+                          </div>
+                        </th>
+                        <th
+                          colspan="1"
+                          role="columnheader"
+                          scope="col"
+                          style="box-sizing: border-box; flex: 150 0 auto; min-width: 30px; width: 150px;"
+                        >
+                          <div
+                            class="px-4 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider flex whitespace-nowrap"
+                          >
+                            Date
+                          </div>
+                        </th>
+                        <th
+                          colspan="1"
+                          role="columnheader"
+                          scope="col"
+                          style="box-sizing: border-box; flex: 150 0 auto; min-width: 30px; width: 150px;"
+                        >
+                          <div
+                            class="px-4 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider flex whitespace-nowrap"
+                          />
+                        </th>
+                      </tr>
+                    </thead>
+                    <tbody
```

Comment:
I think you have to check that the snapshot corresponds to the reality. Here the dashboard is rendered without anything in it.

Reply:
OKay.